frontend/src/cards/ui/UnknownsAlert.tsx

```diff
@@ -130,8 +130,9 @@ function UnknownsAlert(props: UnknownsAlertProps) {
             ? RACE_OR_ETHNICITY
             : breakdownVarDisplayName}
           {/* Age Adjusted Card reports both unknown RACE + AGE */}
-          {secondaryAgePercentageUnknown &&
-            `, and ${secondaryAgePercentageUnknown}${props.metricConfig.shortLabel} reported an unknown age`}
+          {secondaryAgePercentageUnknown
+            ? `, and ${secondaryAgePercentageUnknown}${props.metricConfig.shortLabel} reported an unknown age`
+            : null}
```

Comment:
sweet. Ya We've run into this a couple times; i tend to default to the short circuit operators `&&` but we DEF can't use those in these conditional text blocks. Thanks for the quick fix!!